utils.py

```diff
@@ -977,6 +982,35 @@ def get_latest_folder(folder_path):
     return latest
 
 
+def get_latest_release_url(releases_url):
+    appimage_url = None
+    json_data = json.loads(net_get(releases_url))
+    logging.debug(f"{json_data=}")
+    if json_data is not None:
+        appimage_url = json_data[0].get('assets')[0].get('browser_download_url')  # noqa: E501
+    else:
+        logging.critical("Could not get latest appimage URL.")
+    logging.info(f"Recommended AppImage URL: {appimage_url}")
+    return appimage_url
```

Comment:
Could you make these var names more generic, since we may not always be seeking an AppImage?